manifests/output.pp

```diff
@@ -0,0 +1,35 @@
+# == Define: telegraf::output
+#
+# A Puppet wrapper for discrete Telegraf output files
+#
+# === Parameters
+#
+# [*options*]
+#   Hash. Plugin options for use the the output template.
+#
+# [*sections*]
+#   Hash. Some outputs take multiple sections.
+#
+define telegraf::output (
+  $plugin_type = $name,
+  $options     = undef,
+  $suboptions  = undef,
+) {
+  include telegraf
+
+  if $options {
+    validate_hash($options)
+  }
+
+  if $sections {
+    validate_hash($sections)
+  }
+
+  Class['::telegraf::config']
+  ->
```

Comment:
If you move these chaining arrows (lines 29 & 33) to prepend the same line as their chained directives (e.g. `->\nfile {...` becomes `-> file {...`), I think tests will pass on this PR.